resource/closecaption_english.txt

```diff
@@ -3,4765 +3,4765 @@
 
 
-"vegas.Biomass2"	"<clr:84,255,159>Vegas: Biomass! Man, I hate this stuff. Where is the damn flamer?"
+"vegas.Biomass2"	"<clr:84,255,159><B>Vegas:<B> Biomass! Man, I hate this stuff. Where is the damn flamer?"
 
 
-"vegas.Biomass3"	"<clr:84,255,159>Vegas: Bleagh! Swarm biomass. Anyone got a flamer?"
+"vegas.Biomass3"	"<clr:84,255,159><B>Vegas:<B> Bleagh! Swarm biomass. Anyone got a flamer?"
 
 
-"vegas.Biomass4"	"<clr:84,255,159>Vegas: Hmmmm, large swarm biomass. Anyone got a flamer?"
+"vegas.Biomass4"	"<clr:84,255,159><B>Vegas:<B> Hmmmm, large swarm biomass. Anyone got a flamer?"
 
 
-"vegas.Biomass5"	"<clr:84,255,159>Vegas: Swarm biomass! Anyone got a flamer?"
+"vegas.Biomass5"	"<clr:84,255,159><B>Vegas:<B> Swarm biomass! Anyone got a flamer?"
 
 
-"vegas.Biomass6"	"<clr:84,255,159>Vegas: Urgh! More goo! Let's set it on fire!"
+"vegas.Biomass6"	"<clr:84,255,159><B>Vegas:<B> Urgh! More goo! Let's set it on fire!"
 
 
-"vegas.Biomass7"	"<clr:84,255,159>Vegas: Urgh! Swarm biomass. Where's the flamer?"
+"vegas.Biomass7"	"<clr:84,255,159><B>Vegas:<B> Urgh! Swarm biomass. Where's the flamer?"
 
 
-"vegas.TimeToLeave0"	"<clr:84,255,159>Vegas: Objective completed. Let's hit the slots."
+"vegas.TimeToLeave0"	"<clr:84,255,159><B>Vegas:<B> Objective completed. Let's hit the slots."
 
 
-"vegas.TimeToLeave1"	"<clr:84,255,159>Vegas: Sweet! Tequila, here I come."
+"vegas.TimeToLeave1"	"<clr:84,255,159><B>Vegas:<B> Sweet! Tequila, here I come."
 
 
-"vegas.WatchOut0"	"<clr:84,255,159>Vegas: Lookout!"
+"vegas.WatchOut0"	"<clr:84,255,159><B>Vegas:<B> Lookout!"
 
 
-"vegas.WatchOut1"	"<clr:84,255,159>Vegas: There!"
+"vegas.WatchOut1"	"<clr:84,255,159><B>Vegas:<B> There!"
 
 
-"vegas.WatchOut2"	"<clr:84,255,159>Vegas: Watchout!"
+"vegas.WatchOut2"	"<clr:84,255,159><B>Vegas:<B> Watchout!"
 
 
-"vegas.Shieldbug1"	"<clr:84,255,159>Vegas: Holy shit."
+"vegas.Shieldbug1"	"<clr:84,255,159><B>Vegas:<B> Holy shit."
 
 
-"vegas.Shieldbug2"	"<clr:84,255,159>Vegas: Madre de dios!"
+"vegas.Shieldbug2"	"<clr:84,255,159><B>Vegas:<B> <I>Madre de dios!"
 
 
-"vegas.Shieldbug3"	"<clr:84,255,159>Vegas: Shit, a shieldbug!"
+"vegas.Shieldbug3"	"<clr:84,255,159><B>Vegas:<B> Shit, a shieldbug!"
 
 
-"vegas.ShieldbugHint0"	"<clr:84,255,159>Vegas: They're vulnerable from behind."
+"vegas.ShieldbugHint0"	"<clr:84,255,159><B>Vegas:<B> They're vulnerable from behind."
 
 
-"vegas.ShieldbugHint1"	"<clr:84,255,159>Vegas: Use grenades."
+"vegas.ShieldbugHint1"	"<clr:84,255,159><B>Vegas:<B> Use grenades."
 
 
-"vegas.ShieldbugHint2"	"<clr:84,255,159>Vegas: Who's gonna get behind it?"
+"vegas.ShieldbugHint2"	"<clr:84,255,159><B>Vegas:<B> Who's gonna get behind it?"
 
 
-"vegas.Parasite0"	"<clr:84,255,159>Vegas: Look out, parasite!"
+"vegas.Parasite0"	"<clr:84,255,159><B>Vegas:<B> Look out, parasite!"
 
 
-"vegas.Parasite2"	"<clr:84,255,159>Vegas: Site!"
+"vegas.Parasite2"	"<clr:84,255,159><B>Vegas:<B> Parasite!"
```

Comment:
personally I would have made it `'site!` and added a translator note but this works fine too.

Reply:
As I couldn't find any spanish native speaker which knows about such an abbreviation I didn't want to invent it :)

Reply:
I think he's speaking English in this line.

Like how someone might say "fridge" instead of "refrigerator".

Reply:
I see, but this only works with established words. Now the question is: would this be an well know word in their universe? I would say yes. But is it an well known and often used word in the real world? I would say no.
So personally I would like to keep it that way. In-universe: audio-slang for cool Vegas. Out-universe: readable word for human players.